packages/fxa-auth-server/lib/senders/emails/templates/lowRecoveryCodes/en.ftl

```diff
@@ -1,11 +1,18 @@
 # The user has a low number of valid recovery codes remaining for use
-codes-reminder-title = Low recovery codes remaining
-codes-reminder-description = We noticed that you are running low on recovery codes. Please consider generating new codes to avoid getting locked out of your account.
-codes-generate = Generate codes
-codes-generate-plaintext = { codes-generate }:
-lowRecoveryCodes-action = Generate codes
-lowRecoveryCodes-subject =
+codes-reminder-title-zero = You’re out of backup verification codes!
+codes-reminder-title-one = You’re on your last backup verification code
+codes-reminder-title-two = Time to create more backup verification codes
+
+codes-reminder-description-part-one = Backup verification codes help you restore your info when you forget your password.
+codes-reminder-description-part-two = Create new codes now so you don’t lose your data later.
+codes-reminder-description-two-left = You only have two codes left.
+codes-reminder-description-create-codes = Create new backup verification codes to help you get back into your account if you’re locked out.
```

Comment:
```suggestion
codes-reminder-title-zero = You’re out of backup authentication codes!
codes-reminder-title-one = You’re on your last backup authentication code
codes-reminder-title-two = Time to create more backup authentication codes

codes-reminder-description-part-one = Backup authentication codes help you restore your info when you forget your password.
codes-reminder-description-part-two = Create new codes now so you don’t lose your data later.
codes-reminder-description-two-left = You only have two codes left.
codes-reminder-description-create-codes = Create new backup authentication codes to help you get back into your account if you’re locked out.
```

Suggesting authentication per this thread: https://mozilla-hub.atlassian.net/browse/FXA-4423